README.md

```diff
@@ -1 +1,172 @@
+
+The error tells us that there's no such directory and in fact that's true. Our data folder is in `node-modules-path`, and not in its parent folder. One option is to edit the file to reflect the change:
+
+```
+var fs = require('fs')
+var path = require('path')
+var customers = fs.readFileSync(path.join('node-modules-path', 'data', 'customers.csv'), {encoding: 'utf8'})
+console.log(customers)
+```
+
+The script will work from the parent folder now, but I think you see the problem with this approach. Every time we change the folder from which we launch our Node script, we have to modify the source code.
+
+How about hard-coding an absolute path? That will work for now, but there's a potential problems with with hard-coded paths when you deploy an app. The absolute path on your computer will be very different from the production absolute path. This will cause issues.
+
+Obviously, this is not a good way to write programs. A better way is to use a dynamic absolute path, but how do we get it?
+
+To get an absolute path, there are is method `process.cwd()`, and two properties `__dirname` `process.env.PWD`. They are slightly different. We start with the property `__dirname`, because that's what we need to solve this problem.
+
+This lesson will cover how to work with paths, use them when importing modules. We'll also show a code organization technique where you can import folders.
+
+## Working with `__dirname`
+
+`__dirname` returns the global path to the location of the code being executed. Our problem is fixed by changing relative path to the CSV file to an absolute one:
+
+```
+var fs = require('fs')
+var path = require('path')
+var customers = fs.readFileSync(path.join(__dirname, 'data', 'customers.csv'), {encoding: 'utf8'})
+console.log(customers)
+```
+
+Yay! Try to run this code on your own from different locations like:
+
+```
+node customers.js
+node node-modules-path/customers.js
+```
+
+## Using process.cwd()
+
+When it comes to `process.cwd()` we get the current working directory of the process. In other words, it's an absolute path to the location from which we launch our script (the script which has the `process.cwd()`).
+
+In some cases, `process.cwd()` is the same as `__dirname`. It's when we launch the script from the folder in which the script is located, e.g., with `node customers.js` the value returned by `cwd()` is **the same** as the value of `__dirname`.
+
+In any other location, the `cwd()` will be different from the `__dirname` value, e.g., with `node node-modules-path/node customers.js`, the value returned by `cwd()` is **different** from the value of `__dirname`.
```

Comment:
This is pretty good -- it's a little confusing what "in any other location" refers to here (it's about where the command is invoked from -- maybe we can make that more explicit)